docs/user-guide/rules/about.md

```diff
@@ -61,39 +61,39 @@ The first part describes what [_thing_](http://apps.workflower.fi/vocabs/css/en)
 For example:
 
 ```
-"number-leading-zero"
-// ↑          ↑
-// the thing  what the rule is checking
+"shorthand-property-no-redundant-values"
+ ↑                  ↑
+ the thing          what the rule is checking
 ```
 
 There is no first part when the rule applies to the whole stylesheet.
 
 For example:
 
 ```
-"no-eol-whitespace"
-"indentation"
-//    ↑
-// what the rules are checking
+"no-duplicate-selectors"
+"max-nesting-depth"
+ ↑
+ what the rules are checking
 ```
 
-_Rules are named to encourage explicit, rather than implicit, options._ For example, `color-hex-case: "upper"|"lower"` rather than `color-hex-uppercase: "always"|"never"`. As `color-hex-uppercase: "never"` _implies_ always lowercase, whereas `color-hex-case: "lower"` makes it _explicit_.
+_Rules are named to encourage explicit, rather than implicit, options._ For example, `font-weight-notation: "numeric"|"named-where-possible"` rather than `font-weight-numeric: "always"|"never"`. As `font-weight-named: "never"` _implies_ always numeric, whereas `font-weight-notation: "numeric"` makes it _explicit_.
 
 ### No rules
 
 Most rules require _or_ disallow something.
 
 For example, whether numbers _must_ or _must not_ have a leading zero:
 
-- `number-leading-zero`: `string - "always"|"never"`
-  - `"always"` - there _must always_ be a leading zero
-  - `"never"` - there _must never_ be a leading zero
+- `color-named`: `string - "always-where-possible"|"never"`
+  - `"always"` - there _must always (where possible)_ be named colors
+  - `"never"` - there _must never_ be named colors
```

Comment:
[suggest] It seems better to keep consistency with the sentence above ("For example, whether ..."):

```suggestion
  - `"always-where-possible"` - colors _must always (where possible)_ be named
  - `"never"` - colors _must never_ be named
```